plugins/version.py

```diff
@@ -14,7 +15,11 @@ def version(bot, mask, target, args):
     """
 
     try:
-        repo = Repo()
+        """Get path of plugins folder and derive git root"""
+        git_root = os.path.dirname(os.path.realpath(__file__))[:-8]
```

Comment:
Okay, we might just use `os.path.realpath(__file__ + '/../')` to simply jump up one directory. IMHO That's a little more futureproof, e.g. should we ever decide to rename `plugins` to something else.